e2e/iam/project_api_keys_test.go

```diff
@@ -56,6 +56,19 @@ func TestProjectAPIKeys(t *testing.T) {
 		}
 	})
 
+	t.Run("Assign", func(t *testing.T) {
+		cmd := exec.Command(cliPath, iamEntity,
+			projectEntity,
+			apiKeysEntity,
+			"updates",
+			ID,
+			"--role=GROUP_DATA_ACCESS_READ_ONLY",
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		assert.NoError(t, err, string(resp))
```

Comment:
can you please add a deserialisation test as well? check the role you gave it matches?

Reply:
this endpoint doesn't return a response. If you want I can run again `list` and check that `update` was successful

Reply:
Oh yeah, now I remember, let's let it be then